### PR TITLE
Fixes single site RGW suite issue

### PR DESCRIPTION
**Pull request by udaysk23**
Fixes single site RGW suite issue
log of successful run: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-0WK183

Signed-off-by: uday kurundwade <ukurundw@redhat.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/rgw/sanity_rgw.py [hunk 1]
```diff
@@ -131,7 +131,8 @@ def run(ceph_cluster, **kw):
     install_common = config.get("install_common", True)
     if install_common:
         if ceph_cluster.rhcs_version.version[0] > 4:
-            setup_cluster_access(ceph_cluster, exec_from)
+            setup_cluster_access(ceph_cluster, rgw_node)
```
Comment (Ameenasuhani) [inline tests/rgw/sanity_rgw.py:134]: I have a question+suggestion here for improvement
setup_cluster_access() module perform "yum install -y ceph-common --nogpgcheck" 
https://github.com/red-hat-storage/cephci/blob/master/utility/utils.py#L968 then why do we have explicitly redo the same thing in https://github.com/red-hat-storage/cephci/blob/master/tests/rgw/sanity_rgw.py#L135-L137 @ckulal is there anything around parallel module for which this was added?

@udaysk23 and @ckulal If not can we have it removed?

Comment (ckulal) [reply tests/rgw/sanity_rgw.py:134]: @Ameenasuhani , No this is not included as a part of parallel execution,for parallel support we just moved logic within if condition.

**Review by Ameenasuhani — APPROVED** [review]
LGTM

**Review by anrao19 — APPROVED** [review]

**Review by viduship — APPROVED** [review]

**Comment by anrao19** [comment]
Updating this as DNM for now.
Tested this change with reverting changes done in previously merged pr : https://github.com/red-hat-storage/cephci/pull/1913 --> seeing the issue filenotfounderror

**Comment by anrao19** [comment]
Removing DNM, since issue not observed with latest run : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-PRY6TN

**Review by mkasturi18 — APPROVED** [review]

**Pull request merged**